### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Sadhana-Panthi**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Sadhana-Panthi — COMMENTED** [review]
learning

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (sadhana70) [inline index.html:47]:
```suggestion
  </tr>
```

Comment (sadhana70) [inline index.html:47]:
```suggestion
  </tr>
```

Comment (Sadhana-Panthi) [inline index.html]:
```suggestion
  </tr>hiiiiiiii
```

**Pull request merged**